src/familiar/dropFamiliars.ts

```diff
@@ -150,6 +159,43 @@ const rotatingFamiliars: StandardDropFamiliar[] = [
         )) /
       11,
   },
+  {
+    familiar: $familiar`Rockin' Robin`,
+    expected: (i) => (i === 0 ? 30 - rockinRobinProgress : 30),
+    drop: $item`robin's egg`,
+  },
+  {
+    familiar: $familiar`Optimistic Candle`,
+    expected: (i) => (i === 0 ? 30 - optimisticCandleProgress : 30),
+    drop: $item`glob of melted wax`,
+  },
+  {
+    familiar: $familiar`Garbage Fire`,
+    expected: (i) => (i === 0 ? 30 - garbageFireProgress : 30),
+    drop: [
+      $item`burning newspaper`,
+      $item`extra-toasted half sandwich`,
+      $item`mulled hobo wine`,
+    ]
+  },
+  {
+    familiar: $familiar`Shorter-Order Cook`,
+    expected: () => 11, // 9 with blue plate equipped
+    drop: [
+      $item`short stack of pancakes`,
+      $item`short stick of butter`,
+      $item`short glass of water`,
+      $item`short white`,
+      $item`short beer`,
+    ],
+  },
```

Comment:
Didn't notice that file existed!

I think it's more likely this one was constant value because, before I changed it, `expected` was an array (and so not compatible with infinite drops).

Reply:
Actually, this reminds me of a minor (but surmountable) issue with the entire conceit of this PR:

The purpose of separating `constantValueFamiliars` from `dropFamiliars` has to do with the expectedTurns used in barfFamiliar. If, for example, a shorter-order cook is worth more than your hobo monkey, you're never going to use the hobo monkey. So you want to run all of the familiars that are better than the cook, and then run the cook. If nothing that's unlimited is better than your hobo monkey, you want to run the hobo monkey until the last minute, and then switch over to your various drops familiars (because you have more +weight at the start of the day. Especially if you're going to overdrink).

There's kind of a complicating element here, though, and that is that I can easily imagine a situation where those first 5 turns with a garbage fire are worth way more than your hobo monkey, but grinding out drops for 30 turns is worth way less. 

The way we implemented something similar for the cookbookbat is by making it both a dropFamiliar (which accounts for the recipe drop AND the ingredients) _and_ a constantValueFamiliar (which only accounts for the ingredients). I think that's a fine thing to do for one case, but I'm not sure if it's the right thing to do with this many familiars.

Which is to say, we will need to rethink this at least a little bit.